migrations/2022-01-10-175606_giveaway_migration/up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE Giveaways (
+	id BIGINT UNIQUE NOT NULL AUTO_INCREMENT,
+	message_id BIGINT UNSIGNED NOT NULL,
+	start_time DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	end_time DATETIME NOT NULL,
+	max_winners BIGINT NOT NULL,
+	prize TEXT NOT NULL,
+	completed BOOLEAN NOT NULL DEFAULT FALSE,
+	PRIMARY KEY (id)
+);
```

Comment:
The giveaway table doesn't include `channel_id` but the struct in src/models.rs does. plz fix